cdp_scrapers/instances/seattle.py

```diff
@@ -72,14 +72,11 @@ def __init__(self):
         )
 
         try:
+            url = "https://seattlechannel.org/"
             urlopen("https://seattlechannel.org/")
-        except URLError:
```

Comment:
Seems like https://seattlechannel.org corrected their SSL, as this Exception was getting raised during tests.